rewrite-java/src/main/java/org/openrewrite/java/tree/MethodTypeBuilder.java

```diff
@@ -32,36 +54,92 @@
     String name;
     List<Parameter> parameters = new ArrayList<>();
 
+    /**
+     * See {@link MethodTypeBuilder} for usage.
+     *
+     * @return A new builder.
+     */
```

Comment:
This comment seems useless to me

Reply:
So, I put this in there, such that if you are in your IDE and you roll over a reference to the static method, you can see the documentation for the builder: 

![image](https://user-images.githubusercontent.com/1878529/100773650-05d76a00-33b6-11eb-923a-d4e209d5c3e6.png)

Reply:
I am OK with removing it since the builder API is really self-describing.

Reply:
The example is nice. Definitely worth having a comment for.